src/main/java/com/example/notes/controller/NoteController.java

```diff
@@ -22,4 +20,10 @@ NoteDtoList getAllNotes() {
         return noteService.getAllNotes();
     }
 
+    @PostMapping("/create")
+    @ResponseStatus(HttpStatus.CREATED)
+    NoteDto createNote(@RequestBody NoteDto noteDto) {
```

Comment:
**createNote** can be renamed to **create**.

Reply:
done